c2corg_ui/templates/outing/filters.html

```diff
@@ -56,29 +59,26 @@
       </div>
 
       <div class="col-xs-12 col-sm-6 filter">
-        <label translate>lang</label><br>
-        ${add_multiselect('l', default_langs)}
-      </div>
-
-      <div class="col-xs-12 col-sm-6 filter">
-        <label translate>quality</label><br>
-        <app-slider filter="qa" filters-list="filtersCtrl.filters" unit=""
-          values-list="['${"','".join(quality_types) |n}']"></app-slider>
+        <label translate>elevation_max</label><br>
+        <app-slider filter="oalt" filters-list="filtersCtrl.filters" max="8850"></app-slider>
       </div>
 
       <div class="col-xs-12 col-sm-6 filter">
-        <label translate>elevation_max</label><br>
-        <app-slider filter="oalt" filters-list="filtersCtrl.filters" max="8850"></app-slider>
+        <label translate>elevation_access</label><br>
+        <app-slider filter="oparka" filters-list="filtersCtrl.filters" max="5000"></app-slider>
       </div>
 
       <div class="col-xs-12 col-sm-6 filter">
         <label translate>height_diff_up</label><br>
         <app-slider filter="odif" filters-list="filtersCtrl.filters" max="5000"></app-slider>
       </div>
 
-      <div class="col-xs-12 col-sm-6 filter">
-        <label translate>elevation_access</label><br>
-        <app-slider filter="oparka" filters-list="filtersCtrl.filters" max="5000"></app-slider>
+      <div class="col-xs-12 col-sm-6 filter"
+           ng-if="filtersCtrl.filters.act.length
+               && (filtersCtrl.filters.act.indexOf('snow_ice_mixed') !== -1 ||
```

Comment:
shouldn't && be on the previous line?

Reply:
It could. But I have reused the way it is done for instance in https://github.com/c2corg/v6_ui/blob/d5ef793673d790adffcea735525ce304f504b971/c2corg_ui/templates/route/filters.html
I am not sure it is worth making such changes everywhere :P

Reply:
I don't really mind, but i've been told that it is better that way =)

Reply:
I think it is better to be consistent and use the same notation than in the other files. Or we could change all files but I don't think it is worth it.